app/validators/hyrax/collection_membership_validator.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+module Hyrax
+  # validates that the title has at least one title
```

Comment:
This comment should be fixed (with something better than my suggestion :slightly_smiling_face: ).
```suggestion
  # Validates that the record passes the multiple membership checker
```